<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AAxesDialog</class>
 <widget class="QDialog" name="AAxesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pbDummy">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>94</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>591</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTitleProperties">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Text properties</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAccept">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>531</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbApplyTitleTextPropToAllAxes">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>143</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to all axes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledOffset">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ticks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLabelProperties">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>229</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Label properties</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbApplyLableTextPropToAllAxes">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply to all axes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cobTickInsideOutside">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>228</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Inside</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Outside</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inside and outside</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
